OracleWebLogic/samples/12213-webtier-apache/README.md

```diff
@@ -0,0 +1,97 @@
+Apache Web Server with Oracle WebLogic Server Proxy Plugin on Docker
+===============
+This project includes a quick start Dockerfile and samples for standalone Apache Web Server with 12.2.1.3.0 Oracle WebLogic Server Proxy Plugin based on Oracle Linux and Oracle JDK 7 (Server). The certification of Apache on Docker does not require the use of any file presented in this repository. Customers and users are welcome to use them as starters, and customize/tweak, or create from scratch new scripts and Dockerfiles.
+
+## How to Build Apache With Plugin Docker Image
+
+This project offers a Dockerfile for Apache Web Server with Oraacle WebLogic Server Proxy Plugin in standalone mode. To assist in building the images, you can use `buildDockerImage.sh` script. See below for instructions and usage.
+
+The `buildDockerImage.sh` script is just a utility shell script that performs MD5 checks and is an easy way for beginners to get started. Expert users are welcome to directly call docker build with their preferred set of parameters.
+
+IMPORTANT: You have to download the **Oracle WebLogic Server Proxy Plugin 12.2.1.3.0** package (see .download file) and drop them in this directory.
+
+Run `buildDockerImage.sh` script.
+
+        $ sh buildDockerImage.sh 
+
+## How to run container 
+
+Run an Apache container to access an admin server that is running on <host> and listening to <port>.
+
+        $ docker run -d -e WEBLOGIC_HOST=i<host> WEBLOGIC_PORT=<port> -p 80:80 12213-apache
+
+Run an Apache container to access and load balance to a list of managed servers running standalone 
+ 
+        $ docker run -d -e WEBLOGIC_CLUSTER=host1:port,host2:port,host3:port --net=<some net> -p 80:80 12213-apache
+
+Run an Apache image to proxy and load balance to a list of managed servers in a cluster
```

Comment:
How is the example for load balancing to individual or clustered managed server different.  The sample seems exactly the same.

Reply:
It is just list of servers. for cluster there is an option to use a cluster URL.